server/server.go

```diff
@@ -2307,6 +2331,73 @@ func (s *Server) v1StatusGet(w http.ResponseWriter, r *http.Request) {
 	writer.JSON(w, http.StatusOK, resp, pretty)
 }
 
+func (s *Server) validateInput(input interface{}) []types.BadRequestErr {
+
+	info, err := os.Stat(s.schemaPath)
+	if err != nil {
+		return []types.BadRequestErr{types.BadRequestErr(err.Error())}
+	}
+
+	if !info.IsDir() {
+		bs, err := os.ReadFile(s.schemaPath)
```

Comment:
Does this cache the file in memory? If not, perhaps we should consider doing that ourselves, as it'll impact performance to read files on every request.

As a user, I don't think I'd be very upset if files were only read on startup, and changes on disk weren't picked up during runtime. But if that's too strict, I bet it's possible to set up some sort of file monitor.

Reply:
Good point. I think we can start with reading the file once at startup and in the future we can add more capabilities like adding a file watcher.

Reply:
Done